travis-docker-compose.yml

```diff
@@ -23,13 +28,38 @@ services:
       MICROMASTERS_SECURE_SSL_REDIRECT: 'False'
       MICROMASTERS_DB_DISABLE_SSL: 'True'
       ELASTICSEARCH_URL: elastic:9200
+      BROKER_URL: redis://redis:6379/4
+      CELERY_RESULT_BACKEND: redis://redis:6379/4
     env_file: .env
     ports:
       - "8079:8079"
     links:
       - db
       - elastic
 
+  celery:
```

Comment:
Which tests do we run using an instance of Celery? I remember talking to you about this earlier but it seems like  `CELERY_ALWAYS_EAGER` is always true, so I'm not sure we're actually using Celery in any of the tests